controller/login.go

```diff
@@ -53,9 +62,10 @@ func (c *LoginController) Callback(ctx *app.CallbackLoginContext) error {
 	redirectURL := rest.AbsoluteURL(ctx.RequestData, client.CallbackLoginPath(), nil)
 	redirectTo, err := c.app.AuthenticationProviderService().LoginCallback(ctx, state, code, redirectURL)
 	if err != nil {
-		ctx.ResponseData.Header().Set("Location", redirectURL+"?error="+err.Error())
-		return ctx.TemporaryRedirect()
+		errorPage := c.config.GetErrorPageRedirect()
+		ctx.ResponseData.Header().Set("Location", fmt.Sprintf("%s?error=%v", errorPage, err.Error()))
```

Comment:
We should be using https://github.com/fabric8-services/fabric8-auth/blob/master/rest/rest.go#L126
It will work even if the errorPage already has some params. It will also encode the error param if needed.

BTW it's worth moving that AddParam() to common :) In a separate PR.